src/centralserver/admin-service/src/main/java/org/niis/xroad/centralserver/restapi/config/CurrentHAConfigStatus.java

```diff
@@ -0,0 +1,56 @@
+/**
+ * The MIT License
+ * <p>
+ * Copyright (c) 2019- Nordic Institute for Interoperability Solutions (NIIS)
+ * Copyright (c) 2018 Estonian Information System Authority (RIA),
+ * Nordic Institute for Interoperability Solutions (NIIS), Population Register Centre (VRK)
+ * Copyright (c) 2015-2017 Estonian Information System Authority (RIA), Population Register Centre (VRK)
+ * <p>
```

Comment:
License block text should not be modified (disable  'generate `<p>` on empty lines' in javadoc code style settings)

Reply:
fixed: 
2e7265c